src/cljs/harja/ui/ikonit.cljs

```diff
@@ -615,6 +615,29 @@
 (defn nelio-info []
   [:img {:src "images/harja-icons/status/info.svg" :alt "info"}])
 
+(defn navigation-up []
+  [:img.navigation-up {:src "images/harja-icons/navigation/up.svg" :alt "up"}])
+(defn navigation-right []
+  [:img.navigation-right {:src "images/harja-icons/navigation/right.svg" :alt "right"}])
+(defn navigation-down []
+  [:img.navigation-down {:src "images/harja-icons/navigation/down.svg" :alt "down"}])
+(defn navigation-left []
+  [:img.navigation-left {:src "images/harja-icons/navigation/left.svg" :alt "left"}])
+
+(defn navigation-ympyrassa [suunta]
+  [:div.navigation-ympyrassa
+   {:style {:background-color "#E6F0FA"
```

Comment:
Tämän tyylin vois siirtää johonkin less tiedostoon ja ottaa bgcolorin muuttujasta.

Reply:
Jes, teenpä tän

Reply:
Tää oli itse asiassa tosi hyvä muutos, kiitti vielä kun bongasit

Reply:
Siellä nimittäin oli valmiina jo lessissäkin .navigation ympyrässä tyylejä, niin olis jäänyt kahteen paikkaan. nyt nätisti nipussa.